about/changelog.md

```diff
@@ -8,6 +8,24 @@ keywords: [changelog, upgrades, updates, releases]
 
 All the latest features and updates to Timescale products.
 
+## 🎃 New AI, data integration, and performance enhancements
+<Label type="date">October 31, 2024</Label>
+
+### Pgai Vectorizer: Vector embeddings as database indexes (Early access)
+Automatically create, update, and maintain embeddings as your data changes. Just like an index, Timescale handles all the complexity: syncing, versioning, and cleanup happen automatically.
+This means no manual tracking, zero maintenance burden, and the freedom to rapidly experiment with different embedding models and chunking strategies without building new pipelines.
+Navigate to the AI tab in your service overview and follow the instructions to add your OpenAI API key and set up your first vectorizer.
+
+![Vectorizer setup](https://s3.amazonaws.com/assets.timescale.com/docs/images/vectorizer-setup.png)
+
```

Comment:
I'd add a point here saying something along the lines of: 

Want to code vectorizers into your database by hand? Follow our [Vectorizer automation guide](https://github.com/timescale/pgai/blob/main/docs/vectorizer.md).